### ISSUE TITLE
multiple en-US wiki pages of same topic

### ISSUE THREAD

**Issue by leela52452** [mdn/sprints#2318]
# Request type

- [ ] Please close this issue, I accidentally submitted it without adding any details
- [ ] New documentation
- [x] Correction or update

# Details

for some reason there is more than one wiki page for a topic.
please look into the issue.

thunderbird screenshot :

![Thunderbird](https://user-images.githubusercontent.com/52065244/67418265-0221c300-f5ba-11e9-8ace-29366a328eca.png)

**Comment by leela52452** [comment]
**see also**
#2321
#2324

**Comment by leela52452** [comment]
`xpidl`

![xpidl](https://user-images.githubusercontent.com/52065244/67634527-1a912680-f8b5-11e9-8ce7-b6267ce214e8.png)

**Comment by leela52452** [comment]
`gecko`

![gecko](https://user-images.githubusercontent.com/52065244/67634574-cfc3de80-f8b5-11e9-9229-adb3a984b929.png)

**Comment by leela52452** [comment]
`extensions`

![extensions](https://user-images.githubusercontent.com/52065244/67634593-06015e00-f8b6-11e9-8860-97e6b095df41.png)

**Comment by leela52452** [comment]
`accessible`

![Screenshot_2019-10-27-22-23-28](https://user-images.githubusercontent.com/52065244/67638576-f0a02a00-f8dd-11e9-8aa5-18cd5587a2bb.png)

**Comment by leela52452** [comment]
`storage`

![storage](https://user-images.githubusercontent.com/52065244/67837311-ae960480-fae6-11e9-9eae-70c807ea0b77.png)

**Comment by leela52452** [comment]
`RSS`

![RSS](https://user-images.githubusercontent.com/52065244/67912066-9e326800-fb80-11e9-8257-bc278ce9855a.png)

**Comment by ExE-Boss** [comment]
Most of these are copies in the old `User:` scope from the **MediaWiki** years.

**Comment by SphinxKnight** [comment]
@leela52452 there is no unicity constraints on the __title__ a page can have on MDN (meaning there can be 5 pages with the same title (e.g. "JavaScript").
However there is, _of course_, a unicity constraint on the slug/URL a page can have.

Those cases tend to happen for good and bad reasons alike.
Good ones:
* A theme/topic with different entry points (e.g. the "JavaScript" section landing page, the "JavaScript" glossary page, the "JavaScript" page in the Learning Area): all of them have their reason to exist as such
* A simple homonymy (ex. `font-weight` is both a CSS property and a CSS descriptor)
Bad ones:
* Copies of the old `User:` scope as @ExE-Boss explained
* Draft/Abandoned content

In other words:
* Please check the URL field after selecting one of the option


That being written, I'd think that, __content-wise__, this issue should be closed as is. Opening an issue on https://github.com/mozilla/kuma to suggest an enhancement of the UI might be another step.

If you encounter pages that need deletion (and would help reduce those lists), please open another issue on this repo so that we can make a call.

**Comment by chrisdavidmills** [comment]
Closing this; there is nothing to do here really.

**Comment by ExE-Boss** [comment]
It&nbsp;might be&nbsp;a&nbsp;good&nbsp;idea to&nbsp;at&nbsp;least filter&nbsp;out&nbsp;pages in&nbsp;the&nbsp;old&nbsp;`User:`&nbsp;scope, like&nbsp;what&nbsp;**search** and&nbsp;[`top‑level`](https://developer.mozilla.org/docs/top-level)&nbsp;does.
